npm/css-src/sass/_header.scss

```diff
@@ -1,8 +1,61 @@
+$govuk-header-link: govuk-colour("white");
+
 .govuk-header__tna-logo {
-    width: 33.33%;
+    width: 220px;
     float: left;
 }
 
 .govuk-header__tna-logo-image {
     width: 220px;
-}
\ No newline at end of file
+}
+
+.govuk-header__tna-content {
+    width: 290px;
+    position: relative;
+    margin: 0 auto;
+}
+
+.govuk-header__tna_link {
+    text-decoration: none;
+
+    &:link,
+    &:visited {
+      color: $govuk-header-link;
+    }
+
+    &:focus {
+      @include govuk-focused-text;
+    }
+
+    // alphagov/govuk_template includes a specific a:link:focus selector
+    // designed to make unvisited links a slightly darker blue when focussed, so
+    // we need to override the text colour for that combination of selectors.
+    @include govuk-compatibility(govuk_template) {
+      &:link:focus {
+        @include govuk-text-colour;
+      }
+    }
+  }
+
+  .govuk-header__tna_link--homepage {
+    // Font size needs to be set on the link so that the box sizing is correct
+    // in Firefox
+    @include govuk-font($size: false, $weight: bold);
+
+    display: inline-block;
+    font-size: 30px; // We don't have a mixin that produces 30px font size
+    line-height: 1;
+
+    &:link,
+    &:visited {
+      text-decoration: none;
+    }
+
+    &:active {
+      box-sizing: border-box;
```

Comment:
Great, I'm glad to see this simplifies the styling. 👍 

I've just noticed that we've got a top-level box-sizing rule in [_base.scss](https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/npm/css-src/sass/overrides/_base.scss), but it's not being applied to every element.

It would be good to change it to the more general:

```
html {
  box-sizing: border-box;
}
*, *:before, *:after {
  box-sizing: inherit;
}
```

If you do that, I think you can remove the `box-sizing` lines from this file.

I don't think this affects the elements styled with the GOV.UK design system, but you might want to check the progress bar still looks how you expect.

Reply:
Awesome, I tried putting box-sizing at top level and it seems to be working fine and made no visible changes, the progress bar looks intact too. 😄